lib/compiler/src/compiler.rs

```diff
@@ -16,6 +17,17 @@ use wasmer_types::entity::PrimaryMap;
 use wasmer_types::{Features, FunctionIndex, LocalFunctionIndex, SignatureIndex};
 use wasmparser::{Validator, WasmFeatures};
 
+/// The output of the experimental native compilation.
+/// It returns the object files that the linker will link together after
+/// along with the frame infos necessary for fully traps support.
+#[derive(Clone, Debug, PartialEq, Eq)]
+pub struct ExperimentalNativeCompilation {
+    /// The contents of emitted object files
+    pub object_files: Vec<Vec<u8>>,
```

Comment:
Yeah, the order is deterministic as the current implementation (per llvm-compilation), but IMHO should not be required since it's just an implementation detail?

Reply:
Please comment that the order is irrelevant but deterministic. It helps the next person understand this data.